src/test/routes/comment.spec.js

```diff
@@ -0,0 +1,220 @@
+import {
+  app, chai, expect, sinon, BACKEND_BASE_URL,
+} from '../testHelpers/config';
+import mockData from '../mockData';
+import models from '../../models';
+import authHelper from '../../utils/authHelper';
+
+const { Comment, Request, User } = models;
+const { generateToken } = authHelper;
+const { commentMock } = mockData;
+const {
+  managerId, unAuthorizedManagerId, requesterId, unAuthorizedRequesterId,
+  superAdminId, requestId, invalidRequestId, content, invalidContent, requestWithNoCommentId,
+} = commentMock;
+
+
+describe('COMMENTS', () => {
+  let managerToken, unAuthorizedManagerToken, requesterToken,
+    unAuthorizedRequesterToken, superAdminToken;
+
+  before(async () => {
+    managerToken = generateToken({ id: managerId });
+    unAuthorizedManagerToken = generateToken({ id: unAuthorizedManagerId });
+    requesterToken = generateToken({ id: requesterId });
+    unAuthorizedRequesterToken = generateToken({ id: unAuthorizedRequesterId });
+    superAdminToken = generateToken({ id: superAdminId });
+  });
+
+  const endpoint = `${BACKEND_BASE_URL}/comments`;
+
+  describe('POST /comments/:requestId', () => {
+    it('should return validation error', (done) => {
+      chai
+        .request(app)
+        .post(`${endpoint}/${requestId}`)
+        .set('authorization', managerToken)
+        .send(invalidContent)
+        .end((err, res) => {
+          expect(res.status).to.equal(400);
+          expect(res.body).to.have.property('status').that.equal('error');
+          done(err);
+        });
+    });
+
+    it('should return internal server error', (done) => {
```

Comment:
why should it return `internal server error`
I think the tests should be more descriptive.

Reply:
Thank you for the feedback, I will make them more descriptive.

Reply:
Done. Please resolve the conversation